src/SprykerEco/Zed/Payone/Business/Distributor/PriceDistributor.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Distributor;
+
+use Generated\Shared\Transfer\OrderTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+
+class PriceDistributor implements PriceDistributorInterface
+{
+    protected const PAYMENT_PROVIDER_NAME = 'Payone';
+
+    /**
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     * @return \Generated\Shared\Transfer\OrderTransfer
+     */
+    public function distribute(OrderTransfer $orderTransfer): OrderTransfer
```

Comment:
Let's rename to the `distributeOrderPrice()`.

Reply:
Fixed